doc/classes/ResourceFormatLoader.xml

```diff
@@ -57,6 +57,7 @@
 			<return type="int" />
 			<param index="0" name="path" type="String" />
 			<description>
+				Implement for resource loaders that provide UIDs in a customized way. Leave alone for ".uid" files to be generated along the resource file.
```

Comment:
Not sure about this now, information-wise. Feel free to contest.
Some of the wording is to be consistent with the rest, even if I don't necessarily agree with it myself.
```suggestion
				Should return the unique ID for the resource associated with the given path. If this method is not overridden, an [code].uid[/code] file is generated along with the resource file, containing the unique ID.
```